src/components/ResumeBuilder/Error.js

```diff
@@ -1,5 +1,7 @@
 import React from 'react'
 import { useRouter } from 'next/router'
+import '../../assets/css/resumebuilder.css'
```

Comment:
+1 on import statement

Reply:
I'm not sure what you mean by, '+1 on import statement'

Reply:
The import is not needed

Reply:
Same as https://github.com/Vets-Who-Code/vets-who-code-app/pull/421#discussion_r950646577